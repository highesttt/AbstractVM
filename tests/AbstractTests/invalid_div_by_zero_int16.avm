push int16(1)
push int16(0)
div
exit
