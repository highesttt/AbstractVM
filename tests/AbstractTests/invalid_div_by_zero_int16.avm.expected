Return: 84
Error: [1;31mline 3:[1;33m Division by zero
